src/components/Timeline.js

```diff
@@ -130,7 +130,7 @@ function Timeline() {
           sx={{
             width: 22,
             '& path': {
-              fill: ({ palette }) => palette.primary.main,
+              fill: ({ palette }) => palette.primaryLight.main,
```

Comment:
Primary light does not have a good contrast in the light mode, the `primary` works just fine. Could you change this back?

Reply:
Thanks for the feedback. I have updated in the latest commit.